### ISSUE TITLE
Remove positional and level information for blocks in screen reader output

### ISSUE THREAD

**Issue by BenHenning** [RaspberryPiFoundation/blockly#9395]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Problem

Discussion this week suggested that screen reader output should be simplified and not need positional information.

Separately, `treeitem`s have caused compatibility issues on Safari with VoiceOver.

### Request

Remove `Workspace`'s `treeitem` usage for blocks, though the version for `Flyout` can maybe be kept (more thought is needed there).
